mobile/pubspec.yaml

```diff
@@ -36,6 +36,10 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
 
+  openapi:
+    path: ./gen/
+  dio: '>=4.0.1 <5.0.0'
```

Comment:
`sh ./app_run` でパッケージの依存性エラー出たんだけど太田くんの環境ではどう？

```bash
sh ./app_run.sh
Running "flutter pub get" in mobile...
Resolving dependencies...
Because mobile depends on openapi from path which depends on dio ^5.0.0, dio ^5.0.0 is required.
So, because mobile depends on dio ^4.0.1, version solving failed.
pub get failed
command: "/Users/seiichiro.miyazaki/bin/flutter/bin/cache/dart-sdk/bin/dart __deprecated_pub --color --directory . get
--example"
pub env: {
  "FLUTTER_ROOT": "/Users/seiichiro.miyazaki/bin/flutter",
  "PUB_ENVIRONMENT": "flutter_cli:verify:run",
  "PUB_CACHE": "/Users/seiichiro.miyazaki/.pub-cache",
}
exit code: 1
```

エラーに出てる通り `dio: '^5.0.0'` で指定したら動きました。

Reply:
`sh swagger_gen.sh` した時に gen/pubspec.yaml の dio のバージョンが上がってるっぽい。。
おま環かもしれないのでちょっとこっちで見てみます🙇

Reply:
@zackey1514 
調査なうだったらごめん。
実装時の太田の環境では問題なかったけど、思い当たる点があります！

swagger_gen.sh 内で docker compose してて 
そこで openapi の docker image 使ってるんだけど、
その image の ver指定 を latest にしてるのでそれが原因かも。
([docker-compose.yml](https://github.com/ottaaa/merch/blob/14be22c29219468be2b0ec771153c7547ac1c097/mobile/docker-compose.yml)参照)

openapi の ver固定した方がいいかなって感じなので修正します!

Reply:
太田実装時の openapi の docker image のバージョンが古いのが原因でした。
あと、dio を本体側では使ってないので依存関係を削除して対応してます！
[7d48a91](https://github.com/ottaaa/merch/pull/31/commits/7d48a916dfa083ea7c767811771ee1c4f5cb6f4e)

念の為宮崎君の環境で app_run.sh が動くか確認してほしいです:pray:

Reply:
@ottaaa 
遅くなりました！動作確認できました🙏 ありがとう！